### PR TITLE
[FEATURE] Create a basic console command example

### LINKED ISSUE #119: Create a basic console command example

### PR DESCRIPTION
**Pull request by linawolf**
This is a basic console command as it would be created by "Make"

resolves #119

### CODE REVIEW

File: Classes/Command/DoSomethingCommand.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DoSomethingCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this->setDescription('A Command that does nothing and always fails');
```
Comment (brotkrueml) [inline Classes/Command/DoSomethingCommand.php:15]: Setting the description is superfluous and redundant to the definition in Services.yaml. Maybe we can use setHelp() to preserve the configure() method.

File: Classes/Command/DoSomethingCommand.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // Do awesome stuff
+        return Command::SUCCESS;
```
Comment (brotkrueml) [inline Classes/Command/DoSomethingCommand.php:21]: This does not match with the description. It succeeds, not fails.

File: Configuration/Services.yaml [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
-           identifier: 'form-framework/set-data-structure-identifier'
-           method: 'setDataStructureIdentifier'
-         - name: event.listener
-           identifier: 'form-framework/modify-data-structure-identifier'
-           method: 'modifyDataStructureIdentifier'
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+  T3docs\Examples\:
```
Comment (brotkrueml) [inline Configuration/Services.yaml:6]: I propose to preserve the empty lines between the defintions for better readability.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Classes/Command/DoSomethingCommand.php (hunk 1 above)
Comment (linawolf) [reply Classes/Command/DoSomethingCommand.php]: Done

File: Classes/Command/DoSomethingCommand.php (hunk 2 above)
Comment (linawolf) [reply Classes/Command/DoSomethingCommand.php:21]: Changed the description accordingly

File: Configuration/Services.yaml (hunk 3 above)
Comment (linawolf) [reply Configuration/Services.yaml:6]: I agree, it seems like make removes the whitespaces, which is not nice

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**